Sluggo/View Controllers/TicketDetailViewController.swift

```diff
@@ -48,7 +48,7 @@ class TicketDetailViewController: UIViewController, UITextViewDelegate, UIPicker
         
 
         let memberManager = MemberManager(identity: self.identity)
-        memberManager.listTeamMembers(){ result in
+        memberManager.listFromTeams(page: 1){ (result: Result<PaginatedList<MemberRecord>, Error>) -> Void in
```

Comment:
Not sure what the page is doing with this more generic naming scheme. This seems a bit more difficult to understand what's going on, but maybe I'm missing something.

Reply:
members are paginated so we should be specifying a page, instead of just *assuming* it will be the first